apps/web/src/components/UserProfile/UserProfile.tsx

```diff
@@ -130,427 +43,227 @@ const UserProfile = ({
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.LANGUAGE}>
+                {intl.formatMessage(navigationMessages.languageInformation)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.GOVERNMENT}>
+                {intl.formatMessage(navigationMessages.governmentInformation)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.WORK_LOCATION}>
+                {intl.formatMessage(navigationMessages.workLocation)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.WORK_PREFERENCES}>
+                {intl.formatMessage(navigationMessages.workPreferences)}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink
+                id={PAGE_SECTION_ID.CAREER_TIMELINE_AND_RECRUITMENT}
+              >
+                {intl.formatMessage(
+                  navigationMessages.careerTimelineAndRecruitment,
+                )}
+              </TableOfContents.AnchorLink>
+            </TableOfContents.ListItem>
+            <TableOfContents.ListItem>
+              <TableOfContents.AnchorLink id={PAGE_SECTION_ID.SKILL_SHOWCASE}>
+                {intl.formatMessage(navigationMessages.skillShowcase)}
+              </TableOfContents.AnchorLink>
             </TableOfContents.ListItem>
-          )} */}
           </TableOfContents.List>
         </TableOfContents.Navigation>
-      )}
-      <TableOfContents.Content>
-        {subTitle}
-        {showSection("about") && (
+        <TableOfContents.Content>
           <TableOfContents.Section id={PAGE_SECTION_ID.ABOUT}>
-            <HeadingWrapper show={!!sections.about?.editUrl}>
-              <div
-                data-h2-flex-item="base(1of1) p-tablet(fill)"
-                data-h2-text-align="base(center) p-tablet(left)"
-              >
-                <TableOfContents.Heading
-                  as={headingLevel}
-                  icon={UserIcon}
-                  data-h2-margin-top="base(0)"
+            <div data-h2-padding="base(x2, 0, x1, 0)">
+              <div data-h2-flex-grid="base(center, x2, x1)">
+                <div
+                  data-h2-flex-item="base(1of1) p-tablet(fill)"
+                  data-h2-text-align="base(center) p-tablet(left)"
                 >
-                  {intl.formatMessage(navigationMessages.aboutMe)}
-                </TableOfContents.Heading>
+                  <TableOfContents.Heading
+                    as={headingLevel}
+                    icon={UserIcon}
+                    data-h2-margin-top="base(0)"
```

Comment:
Leave this but also add `data-h2-margin-bottom="base(x1)"` to all `TableOfContents.Heading`